lib/Controller/ConfigController.php

```diff
@@ -140,8 +140,7 @@ public function oauthRedirect(string $code = '', string $state = ''): RedirectRe
 		if ($clientID and $clientSecret and $configState !== '' and $configState === $state) {
 			$redirect_uri = $this->config->getUserValue($this->userId, Application::APP_ID, 'redirect_uri');
 			$adminOauthUrl = $this->config->getAppValue(Application::APP_ID, 'oauth_instance_url', Application::DEFAULT_GITLAB_URL) ?: Application::DEFAULT_GITLAB_URL;
-			$gitlabUrl = $this->config->getUserValue($this->userId, Application::APP_ID, 'url', $adminOauthUrl) ?: $adminOauthUrl;
-			$result = $this->gitlabAPIService->requestOAuthAccessToken($gitlabUrl, [
+			$result = $this->gitlabAPIService->requestOAuthAccessToken($adminOauthUrl, [
```

Comment:
only the admin's oauth url is used here and the personal url is overridden. So what is the use of the `url` config for the user?